tests/integ/test_stored_procedure.py

```diff
@@ -1959,7 +1959,7 @@ def artifact_repo_test(_):
 @pytest.mark.parametrize(
     "version_override, expect_warning",
     [
-        ("1.26.1", False),  # Bugfix version - no warning
+        ("1.27.1", False),  # Bugfix version - no warning
```

Comment:
this is a niche case where we use newer snowpark to create sproc based upon older snowpark
it collects the cloudpickle version used in the newer snowpark (local env) which older version doesn't support

log a jira https://snowflakecomputing.atlassian.net/browse/SNOW-1951792 to improve